app/waf/package.json

```diff
@@ -66,7 +66,8 @@
     "plural": "^1.1.0",
     "typescript-logging": "^0.6.3",
     "uuid": "^3.3.2",
-    "ip": "^1.1.5"
+    "ip": "^1.1.5",
+    "@network-utils/tcp-ping": "^1.1.0"
```

Comment:
are these package safe?

Reply:
They are both safe. But you remind me a lot. 

After the scan with `snyk`, we found the following vulnerabilities. 
`snyk` scan may need to be added to our CI process. It support github.io integration.
@zhaoqin-github 

```
Analyzing npm dependencies for /Users/zong/GitRepos/zongzw/f5-adcaas-openstack/app/waf
Querying vulnerabilities database...
Tested 248 dependencies for known vulnerabilities, found 1 vulnerabilities, 2 vulnerable paths.

? ✗ High severity vuln found in js-yaml@3.13.0, introduced via @loopback/rest@1.9.1
    Description: Arbitrary Code Execution
    Info: https://snyk.io/vuln/SNYK-JS-JSYAML-174129
    From: @loopback/rest@1.9.1 > js-yaml@3.13.0

  Remediation options Skip

? ✗ High severity vuln found in js-yaml@3.13.0, introduced via @loopback/rest-explorer@1.1.13
    Description: Arbitrary Code Execution
    Info: https://snyk.io/vuln/SNYK-JS-JSYAML-174129
    From: @loopback/rest-explorer@1.1.13 > @loopback/rest@1.9.1 > js-yaml@3.13.0

  Remediation options Skip
```